### PR TITLE
Pipe 1 - create database with reference samples

### LINKED ISSUE #1: Create database with reference samples
**Success condition :**
 - We have locally a database with approx 2000 samples

### PR DESCRIPTION
**Pull request by Elrendio**
Resolved #1

** What was done **
- can scrape a whole page a results on caliroot, scraping still not been cleaned
- cleaned pair object
- general scraping of caliroots for US9 pairs

### CODE REVIEW

File: engine/server/datasources.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  },
+  "mongodb-dataset": {
+    "host": "localhost",
+    "port": 27017,
+    "database": "dataset",
+    "password": "",
+    "name": "mongodb-dataset",
+    "connector": "mongodb",
+    "user": ""
+  }
```
Comment (Elrendio) [inline engine/server/datasources.json]: Most of these informations should be in ENV variables

File: engine/.gitignore [hunk 2]
```diff
@@ -0,0 +1,19 @@
+*.csv
```
Comment (Elrendio) [inline engine/.gitignore]: This file should be in root file not in engine

File: engine/common/models/pair.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+    var rp = require('request-promise');
+    var cheerio = require('cheerio');
+
+    var calirootsPair = [];
+
+    Pair.scrapePair = function(pairLink) {
+        return rp(pairLink)
+            .then(function(html) {
+                $ = cheerio.load(html);
```
Comment (Elrendio) [inline engine/common/models/pair.js]: Pourquoi $ comme nom de variable et pas just html ?

File: engine/common/models/pair.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
+module.exports = function(Pair) {
```
Comment (Elrendio) [inline engine/common/models/pair.js]: Indentation in this file is 4, guidelines in `engine/.editorconfig` and javascript guidelines ask for 2

File: engine/common/models/pair.js [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+                Pair.create(pairInfo);
+                return pairInfo;
+            })
+    };
+
+    Pair.scrapePage = function(pageLink) {
+        return rp(pageLink)
+            .then(function(html) {
+                $ = cheerio.load(html);
+                var firstPageLinks = [];
```
Comment (Elrendio) [inline engine/common/models/pair.js]: Why firstPageLinks, seems to me it takes all links on given page.

File: engine/server/datasources.json (hunk 1 above)
Comment (totojab) [inline engine/server/datasources.json]: CF issue #7

File: engine/.gitignore (hunk 2 above)
Comment (totojab) [inline engine/.gitignore]: moved

File: engine/common/models/pair.js (hunk 4 above)
Comment (totojab) [inline engine/common/models/pair.js]: done

File: engine/common/models/pair.js (hunk 3 above)
Comment (totojab) [inline engine/common/models/pair.js]: ok

File: engine/common/models/pair.js (hunk 5 above)
Comment (totojab) [inline engine/common/models/pair.js]: changed to pageLinks

**Pull request merged**